exporter/templates/cat1/r3/_record_target.xml

```diff
@@ -35,12 +35,12 @@
         <family>{{escape .Last}}</family>
       </name>
       <administrativeGenderCode code="{{escape .Gender}}" codeSystem="2.16.840.1.113883.5.1" codeSystemName="HL7 AdministrativeGender"/>
-      <birthTime value="{{timeToFormat .Birthdate "20060102"}}"/>
+      <birthTime value="{{timeToFormat .BirthDate "20060102"}}"/>
```

Comment:
Because not all time values can have an unknown state. The IG does not specify the `nullFlavor` possibility for this particular value.